lib/language_pack/helpers/jvm_installer.rb

```diff
@@ -88,4 +92,8 @@ def fetch_env_untar(key)
     fetch_untar(jvm_version)
     true
   end
+
+  def install_pgconfig_jar(url)
+    run("curl --retry 3 -s -L #{url} -o #{@vendor_dir}/jre/lib/ext/pgconfig.jar")
```

Comment:
Could we use a fetcher here instead? That way if we need to update our curl commands we can do it all in one space.